content/en/docs/Tasks/creating-a-catalog.md

```diff
@@ -17,15 +18,62 @@ be made available to install in a cluster. You can make your operator bundle ava
 it to a catalog, packaging the catalog in a container image, and then using that image reference in the `CatalogSource`.
 This image contains all of the metadata required for OLM to manage the lifecycle of all of the operators it contains.
 
-OLM uses a plaintext [file-based catalog][file-based-catalog-spec] format (JSON or YAML) to store these records in a Catalog, and `opm` has tooling
-that helps initialize a Catalog, add new operators into it, and then validate that the catalog is valid. Let's walk
-through a simple example.
+OLM uses a plaintext [file-based catalog][file-based-catalog-spec] format (JSON or YAML) to store these records in a Catalog, and there are two approaches we can take to creating a Catalog, adding operators to it, and validating it. 
+Let's walk through a simple example for both approaches.
 
+### Catalog Creation Using Veneers
+[Veneers][veneers-doc] are a purpose-built simplification of [File-Based Catalogs][file-based-catalog-spec] to ease common catalog operations.  For this example, we'll be using the [semver veneer][semver-veneer-doc].
+>Note: We strongly recommend that authors create and maintain their veneers in a version-controlled environment discrete from their generated catalogs.  Further, we recommend that authors focus on the veneer as the sole artifact connecting the operator to the catalog (even going so far as only generating the file-based catalog during CI/CD tooling so it is only provided for catalog contribution.)
+
+#### Catalog Creation
+First we need to create the Catalog hierarchy and Dockerfile for generating the image
+```sh
+$ mkdir -p cool-catalog/example-operator
+$ opm generate dockerfile cool-catalog
+```
+
+#### Organize the Bundles into Channels
+
+Let's assume that this isn't the first time that we have released this operator into the catalog, but it's our first foray into veneers.  We need to ensure an upgrade graph edge between the older bundle version and the new one.  We also want to promote this latest version in a "stable" channel.  Lastly, we already use [Semantic Versioning](https://semver.org) for our release numbering, and we really only care about new major (e.g. X.\#.\#) releases. 
+
+>Note: we presume this step and veneer processing are performed in the source-controlled location related to operator bundle release, or at least separate from the catalog
+
+```sh
+$ cat << EOF >> example-operator-veneer.yaml
+Schema: olm.semver
+GenerateMajorChannels: true
+GenerateMinorChannels: false
+Stable:
+  Bundles:
+  - Image: repository-uri/example-operator:v0.8.9
+  - Image: repository-uri/example-operator:v0.9.0
+EOF
+```
+
+#### Generate the Catalog
+
+```sh
+$ opm alpha render-veneer semver -o yaml < example-operator-veneer.yaml > cool-catalog/catalog.yaml
+```
+
+Now validate the catalog to ensure that the result is functional
+```sh
+$ opm validate cool-catalog
+$ echo $?
+0
+```
+
+
+### Catalog Creation with Raw File-Based Catalogs
+
+
+
+#### Catalog Creation
```

Comment:
I am not sure of the standard practice in the Kubernetes or OpFrame docs, but I think that if you swiched the subheadings in the procedures to numbered lists it would improve readability and minimalism (ie, some of the smaller steps require a lead-in due to the format. if they were just steps in a procedure without subheadings, the list item could provide all the necessary context)

Reply:
I have no idea how to do this at present, but we'll have to revisit this doc later after alpha and can tackle then.